frontend/src/Components/Elements/Table/Table.js

```diff
@@ -33,6 +34,25 @@ function Table({
       paginated ? usePagination : ''
     );
 
+  const displayMissingPhrase = (ds) => {
+    let phrase = {
+      STOPS: 'stops',
+      SEARCHES: 'searches',
+      STOPS_BY_REASON: 'stops',
+      SEARCHES_BY_TYPE: 'searches',
+      USE_OF_FORCE: 'use of force',
+      CONTRABAND_HIT_RATE: 'contraband',
+      LIKELIHOOD_OF_SEARCH: 'searches',
+    }[ds];
+    if (phrase === undefined && Array.isArray(ds)) {
+      phrase = 'searches';
+    }
+    if (!phrase) {
+      phrase = 'data';
+    }
+    return `No ${phrase} have been reported`;
```

Comment:
Maybe some terms should be has rather than have? Like "No use of force has been reported"

Reply:
Yes, updated.